uast/types.go

```diff
@@ -152,6 +174,11 @@ func NewValue(typ string) (reflect.Value, error) {
 	}
 }
 
+// TypeOf returns the UAST type of a value.
+//
+// If the value is a generic UAST node, the function returns the value of its KeyType.
+//
+// If an object is registered as a UAST schema type, the function returns the associated type.
```

Comment:
Just to help me understand the convention/consistency - when an empty newline shall be added to multiple comments and when not?

Reply:
If I remember correctly, godoc merges lines into a single paragraph if there is no newline in between.